src/Parser/TemplateParser.cs

```diff
@@ -106,11 +107,15 @@ private string ParseFormat()
             // TODO: Escaped }} in formats?
             return ReadUntil('}');
         }
-        private string ParseAlignment()
+
+        private int ParseAlignment()
         {
             Skip(',');
-            //tood don't parse non-numeric?
-            return ReadUntil(AlignmentDelimiters);
+            string text = ReadUntil(AlignmentDelimiters);
+            int alignment;
+            if (!int.TryParse(text, NumberStyles.Integer, CultureInfo.InvariantCulture, out alignment))
```

Comment:
There is no `OrdinalCulture` you confuse with string comparison options. (Code is removed anyway).